tests/test_utils.py

```diff
@@ -1,153 +1,106 @@
-                "attributes": [
-                    {
-                        "name": "Title",
-                        "value": "Cells of the adult human heart"
-                    },
-                    {
-                        "name": "Description",
-                        "value": "Cardiovascular disease is the leading cause of death worldwide."
-                    },
-                    {
-                        "name": "Organism",
-                        "value": "Homo sapiens (human)"
-                    },
-                    {
-                        "name": "alias",
-                        "value": "Phoenix-test-1"
-                    }
-                ],
-                "files": [
-                    {
-                        "path": "test_file.txt",
-                        "attributes": [
-                            {
-                                "name": "Description",
-                                "value": "Raw Data File"
-                            }
-                        ],
-                        "type": "file"
-                    }
-                ],
-                "links": [
-                    {
-                        "url": "ABC123",
-                        "attributes": [
-                            {
-                                "name": "type",
-                                "value": "gen"
-                            }
-                        ]
-                    },
-                    {
-                        "url": "SAMEA7249626",
-                        "attributes": [
-                            {
-                                "name": "Type",
-                                "value": "BioSample"
-                            }
-                        ]
-                    }
-                ],
-                "subsections": [
-                    {
-                        "type": "Author",
-                        "attributes": [
-                            {
-                                "name": "Name",
-                                "value": "John Doe"
-                            }
-                        ]
-                    }
-                ]
-            }
-        }
+        bio_study = TestUtils.create_bio_study_without_file()
+
+        bio_study.section.files = set()
+        bio_study.section.files.add(
+            TestUtils.create_bio_study_file("test_file.txt", "BAM",
+                                            {TestUtils.create_attribute("Description", "Raw Data File")})
+        )
+
+        json_payload = TestUtils.create_json_payload_from_bio_study(bio_study)
+
+        return json_payload
+
+    @staticmethod
+    def create_json_payload_from_bio_study(bio_study):
+        return json.loads(json.dumps(bio_study, cls=EnhancedJSONEncoder))
+
+    @staticmethod
+    def create_bio_study_without_file():
+        bio_study = BioStudy()
+        bio_study.attach_to = "Phoenix Project"
+
+        bio_study.attributes = set()
```

Comment:
My recommendation above would prevent the user of the library from having to do this.

Reply:
Good idea. I will try to do it. Tx

Reply:
Done